### DISCUSSION TITLE
Search bar is not fully visible on S23 ultra

### DISCUSSION THREAD

**Discussion by Abints** [community/community#61036] [Mobile]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I am using Samsung s23 Ultra mobile and when I clicked on the search icon on the top, I am not able to see the search bar fully. Attaching the screenshot here:
![Screenshot_20230717_022905_Chrome](https://github.com/community/community/assets/25296247/0d3cdf1d-a32d-498f-be3b-232e4d124255)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Abints ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by moonfury-ops** [comment]
Thank you
